packages/effector/index.d.ts

```diff
@@ -139,6 +139,8 @@ export interface Effect<Params, Done, Fail = Error> extends Unit<Params> {
   pending: Store<boolean>
   inFlight: Store<number>
   watch(watcher: (payload: Params) => any): Subscription
+  filter(config: {fn(payload: Params): boolean}): Event<Params>
+  filterMap<T>(fn: (payload: Params) => T | undefined): Event<T>
```

Comment:
![image](https://user-images.githubusercontent.com/29141708/90159588-483e5200-dd99-11ea-8da7-21b7c7fb1f1b.png)

Reply:
Is there a reason why you have skipped the first overload?

Reply:
It is forgotten.